2-Draft-Accepted/RFCxxxx-DSC-for-PowerShell7.md

```diff
@@ -0,0 +1,117 @@
+need for OMI ([Open Management Infrastructure](https://github.com/Microsoft/omi)) on Linux.
+This was the start to enable cross platform DSC resources to be authored in PowerShell script (with some limitations).
+
+This feature was shipped as an [experimental feature](https://docs.microsoft.com/powershell/scripting/learn/experimental-features?view=powershell-7) in PowerShell 7.0.
+
+### Bring your own agent
+
+A significant change from existing DSC for Windows PowerShell is there will be no default agent (LCM) shipped with PowerShell 7.
+
+Instead, we want to make it easier to integrate with existing configuration agents like [Ansible](https://docs.ansible.com/ansible/latest/user_guide/windows_dsc.html),
+[Chef](https://docs.chef.io/resources/dsc_resource/), [Puppet](https://forge.puppet.com/puppetlabs/dsc),
+[Azure Policy Guest Configuration](https://docs.microsoft.com/azure/governance/policy/concepts/guest-configuration), and other popular configuration solutions.
+
+Custom agents can simply use the existing PowerShell API (or call out to `pwsh`) to directly invoke DSC resources.
+
+You can even simply write a PowerShell script as your agent that runs off TaskScheduler or cron, for example.
+
+Along with the lack of a default agent, there is no pre-defined push or pull model for deploying DSC.
+
+### Cross platform first
+
+DSC has supported both Windows and Linux, however, the [Linux support](https://docs.microsoft.com/powershell/scripting/dsc/getting-started/lnxGettingStarted?view=powershell-7)
+was dependent on [OMI](https://github.com/Microsoft/omi) and did not support authoring Linux DSC resources in PowerShell script making development
+inconsistent and much more difficult.
+
+DSC for PowerShell 7 will be cross platform having consistent capabilities and user experience on all platforms.
+
+### Moving away from WMI/OMI/MOF
+
+DSC on Windows was originally designed relying on [WMI](https://docs.microsoft.com/windows/win32/wmisdk/wmi-start-page) as a host on Windows,
+OMI as a host on Linux, and [MOF](https://docs.microsoft.com/windows/win32/wmisdk/managed-object-format--mof-) for configuratio and schema.
+
+MOF as a format is not something most developers are familiar with and not much tooling support.
+However, developers of DSC resources written in PowerShell script still needed to learn MOF and also keep their MOF schema
+in sync with their script implementation.
+
+Today, when you execute a [DSC configuration script](https://docs.microsoft.com/powershell/scripting/dsc/configurations/configurations?view=powershell-7),
+the output is a MOF configuration file.
+In general, users do not need to read this MOF file as it's intended to be used by machines.
+However, MOF parsing creates a dependency on WMI/OMI.
+Instead, we will move towards a JSON configuration file format.
+
+Since there is no default agent to understand the JSON configuration file, it will be up to users or 3rd party agents to understand the configuration file.
+
```

Comment:
because of comment above, I'd remove this sentence and JSON config example, that should probably be a different RFC in itself.